survey/sas-local-subnet-candidates.py

```
@@ -0,0 +1,168 @@
+        ip_match = ipv4_re.search(line)
+        if ip_match:
+            value = ip_match.group(1).split("(")[0].strip()
+            pending_ip = value if usable_ipv4(value) else ""
+            continue
+        mask_match = mask_re.search(line)
+        if mask_match and pending_ip:
+            mask = mask_match.group(1).split("(")[0].strip()
+            if usable_ipv4(mask):
+                try:
+                    net = direct_network(pending_ip, mask)
+                except Exception:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)